salt/kube-apiserver/remove-pre-reboot.sls

```diff
@@ -8,6 +8,3 @@
 
 {{ pillar['ssl']['kube_apiserver_key'] }}:
   file.absent
-
-/etc/kubernetes/apiserver:
```

Comment:
This will provide information about K8s cluster configuration including network addresses